hoomd/hpmc/module_convex_polygon.cc

```diff
@@ -45,8 +44,6 @@ void export_convex_polygon(pybind11::module& m)
     export_HarmonicField<ShapeConvexPolygon>(m, "ExternalFieldHarmonicConvexPolygon");
     export_ExternalFieldComposite<ShapeConvexPolygon>(m, "ExternalFieldCompositeConvexPolygon");
     // export_ExternalFieldWall<ShapeConvexPolygon>(m, "WallConvexPolygon");
```

Comment:
The original developers did not implement overlap checks for anisotropic 2D shapes. I don't know why this was commented out instead of raising an error if for unsupported shape–wall combinations. I have fixed that now.